Simplenote/SPAppDelegate.m

```diff
@@ -230,8 +230,6 @@ - (void)setupCrashlytics
     NSString *email = self.simperium.user.email;
     NSString *key = [SPCredentials simplenoteCrashlyticsKey];
```

Comment:
This stays nil for me even after I set `FabricApiKey` and `FabricBuildKey`. Looks like we're still pulling from `simplenoteCrashlyticsKey`. Shouldn't we pull for the fabric key in `SPCredentials.simplenoteCrashlyticsKey()`?

Reply:
I _think_ we still need to have all three keys. The two for fabric are used in the project script strictly to upload the dsym.